pkg/prometheus/statefulset.go

```diff
@@ -350,16 +347,30 @@ func makeStatefulSetSpec(p monitoringv1.Prometheus, c *operator.Config, shard in
 	retentionTimeFlag := "-storage.tsdb.retention="
 	if version.Minor >= 7 {
```

Comment:
Could you also fix this please?
```suggestion
	if version.GTE(semver.MustParse("2.7.0") {
```